src/Symfony/Component/HttpKernel/Controller/ArgumentResolver.php

```diff
@@ -12,6 +12,11 @@
 namespace Symfony\Component\HttpKernel\Controller;
 
 use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Controller\ArgumentValueResolver\DefaultValueResolver;
```

Comment:
That would imply they are `ArgumentResolver` implementations, which they are not, hence I named them a `ValueResolver`. What I could do, is remove it from the namespace as they are tightly coupled to the `ArgumentResolver`.